src/vaadin-contextmenu-items-mixin.html

```diff
@@ -330,7 +330,7 @@
           }
         };
 
-        const openEvent = this._touch ? 'touchstart' : 'mouseover';
+        const openEvent = this._touch ? 'click' : 'mouseover';
```

Comment:
To my experience, touch devices dispatch `mouseover` only when they also dispatch `click`, i. e., if you don’t move your finger while tapping. Could we use `mouseover` opening for both touch and regular devices then, perhaps?

Reply:
Sounds good! Thanks for suggestion.
Updated the PR.